docs/defining-profiles.md

```diff
@@ -0,0 +1,53 @@
+# Defining Profiles
+
+Each profile defines a value for `sourceKey` and `destinationKey`. These keys must extend `string` and should be unique combinations, beyond that the choice is irrelevant to the function of Kitbag Mapper.
+
+Here is a simple example of a profile that converts a python server model you might see in an API service to the frontend model.
```

Comment:
This isn't really python specific and distracts from the example IMO. Feel free to ignore if you think otherwise. 
```suggestion
Here is a simple example of a profile that converts a server model you might see in an API service to the frontend model.
```